plugins/kiro/scripts/hook.ps1

```
@@ -0,0 +1,376 @@
+if (-not $actorEmail) {
+    if ($env:USERNAME -and $env:COMPUTERNAME) { $actorEmail = "$($env:USERNAME)@$($env:COMPUTERNAME)" }
+    elseif ($env:USERNAME) { $actorEmail = $env:USERNAME } else { $actorEmail = $env:COMPUTERNAME }
+}
+
+# -- payload from stdin (recover UTF-8, strip BOM), then the 2.x session id ---
+$payload = [Console]::In.ReadToEnd()
+if (-not $payload) { $payload = '{}' }
+try {
+    $raw = [Console]::InputEncoding.GetBytes($payload)
+    $payload = [System.Text.Encoding]::UTF8.GetString($raw)
+} catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)